scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -302,6 +302,16 @@
   ; to deleting from the map, into (missing,missing) state.
   {:cook-expected-state :cook-expected-state/completed})
 
+(defn handle-pod-failed
+  "Handles pod failure, checking to see if the failure was caused by pod preemption."
+  [{:keys [name] :as compute-cluster} pod-name {:keys [synthesized-state] :as k8s-actual-state-dict}]
+  (if (some-> synthesized-state :pod-preempted-timestamp)
+    (do
+      (log/info "In compute cluster" name ", pod" pod-name
```

Comment:
I"d reword this to be more concrete and clear. Maybe something like:

"has a preemped timestamp and is treated as preempted"

Reply:
Done, let me know what you think

Reply:
lgtm